source/leuven/wice_quick_start.rst

```diff
@@ -74,14 +108,18 @@ The big memory nodes (2048GB of RAM) are also located in a separate partition. I
 
 Submit to a GPU node
 ~~~~~~~~~~~~~~~~~~~~
-The GPU nodes are also located in a separate cluster partition so you will need to explicitly specify it when submitting your job. We also configured the GPU nodes as shared resources, meaning that different users can simultaneously use a portion of the same node. However every user will have exclusive access to the number of GPUs requested. If you want to use only 1 GPU of type A100 you can submit for example like this:
 
-::
+The GPU nodes are located in the ``gpu`` partition, so you will need to explicitly specify 
+it when submitting your job. 
+We also configured the GPU nodes as shared resources, meaning that different users can 
+simultaneously use a portion of the same node. 
```

Comment:
Similar suggestion as for Genius earlier:
```suggestion
Similar to the other nodes, the GPU nodes can be shared by different jobs from different users.
```

Reply:
done